packages/core/src/App/Components/Elements/NotificationMessage/notification.jsx

```diff
@@ -11,6 +11,7 @@ import { BinaryLink } from '../../Routes';
 
 const Notification = ({ data, removeNotificationMessage }) => {
     const { is_dashboard } = React.useContext(PlatformContext);
+    const active_timer = sessionStorage.getItem('popup_timer');
```

Comment:
We shouldn't be maintaining the state outside the component

Reply:
I am using the React ref hook now to call a function inside the child component to remove the timeout session when it's clicked.